apps/omg/lib/omg/state.ex

```diff
@@ -156,7 +154,12 @@ defmodule OMG.State do
   Exits (spends) utxos on child chain, explicitly signals all utxos that have already been spent
   """
   def handle_call({:exit_utxos, utxos}, _from, state) do
-    {:ok, {db_updates, validities}, new_state} = Core.exit_utxos(utxos, state)
+    exiting_utxos = Core.get_exiting_utxo_positions(utxos, state)
+
+    db_utxos = init_utxos_from_db(exiting_utxos, state)
```

Comment:
`init` prefix seems to be used a couple of times... that invalidates the meaning.